target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh

```diff
@@ -35,7 +35,16 @@ redmi_ax6000_initial_setup()
 	fw_setenv flag_boot_success 1
 	fw_setenv flag_try_sys1_failed 8
 	fw_setenv flag_try_sys2_failed 8
-	fw_setenv mtdparts "nmbm0:1024k(bl2),256k(Nvram),256k(Bdata),2048k(factory),2048k(fip),256k(crash),256k(crash_log),30720k(ubi),30720k(ubi1),51200k(overlay)"
+
+        local board=$(board_name)
+        case "$board" in
+        xiaomi,mi-router-wr30u-stock)
+                fw_setenv mtdparts "nmbm0:1024k(bl2),256k(Nvram),256k(Bdata),2048k(factory),2048k(fip),256k(crash),256k>
+                ;;
+        xiaomi,redmi-router-ax6000-stock)
+                fw_setenv mtdparts "nmbm0:1024k(bl2),256k(Nvram),256k(Bdata),2048k(factory),2048k(fip),256k(crash),256k>
```

Comment:
1. may be wrong here.
2. use tab indent.

Reply:
Tab Corrected.

Reply:
mtdparts Corrected.